spec/jobs/segment_identify_job_spec.rb

```diff
@@ -24,10 +24,10 @@
             version: Utils::VersionService.new.version.version.number,
             organization_name: membership.organization.name,
             email: membership.user.email,
-          }
+          },
         )
 
-      subject.perform_now(membership_id: membership_id)
+      subject.perform_now(membership_id:)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)